### PR TITLE
Bump rubygem-little-plugger to 1.1.4-1

### PR DESCRIPTION
**Pull request by evgeni**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.22
* [ ] 1.21
* [ ] 1.20

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: packages/foreman/rubygem-little-plugger/rubygem-little-plugger.spec [hunk 1]
```diff
@@ -1,83 +1,91 @@
...
-Release: 23%{?dist}
+Version: 1.1.4
+Release: 1%{?dist}
+Summary: LittlePlugger is a module that provides Gem based plugin management
 Group: Development/Languages
 License: MIT
-URL: https://rubygems.org/gems/little-plugger
-Source0: http://gems.rubyforge.org/gems/%{gem_name}-%{version}.gem
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
+URL: http://gemcutter.org/gems/little-plugger
```
Comment (ekohl) [inline packages/foreman/rubygem-little-plugger/rubygem-little-plugger.spec]: This redirects now to the URL it used to have.
```suggestion
URL: https://rubygems.org/gems/little-plugger
```

File: packages/foreman/rubygem-little-plugger/rubygem-little-plugger.spec [hunk 2]
```diff
@@ -1,83 +1,91 @@
...
+# template: scl
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name little-plugger
 
-Summary: LittlePlugger is a module that provides Gem based plugin management
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 1.1.3
-Release: 23%{?dist}
```
Comment (ekohl) [inline packages/foreman/rubygem-little-plugger/rubygem-little-plugger.spec]: Wow, 23.

File: packages/foreman/rubygem-little-plugger/rubygem-little-plugger.spec [hunk 3]
```diff
@@ -1,83 +1,91 @@
...
 
 %files
 %dir %{gem_instdir}
 %exclude %{gem_instdir}/.gitignore
-%{gem_instdir}/lib
-%{gem_instdir}/spec
+%{gem_libdir}
 %exclude %{gem_cache}
 %{gem_spec}
-# contains licensing information
```
Comment (ekohl) [inline packages/foreman/rubygem-little-plugger/rubygem-little-plugger.spec]: Is this ok because it's MIT?

Comment (evgeni) [reply packages/foreman/rubygem-little-plugger/rubygem-little-plugger.spec]: probably, but I've reverted that hunk

File: packages/foreman/rubygem-little-plugger/rubygem-little-plugger.spec (hunk 2 above)
Comment (evgeni) [reply packages/foreman/rubygem-little-plugger/rubygem-little-plugger.spec]: Yes.

**Comment by evgeni** [comment]
Updated!

**Pull request merged**